sdk/core/azure-core/azure/core/tracing/ext/opencensus_span.py

```diff
@@ -83,7 +83,10 @@ def to_header(self):
         temp_headers = {}
         if tracer_from_context is not None:
             ctx = tracer_from_context.span_context
-            temp_headers = tracer_from_context.propagator.to_headers(ctx)
+            try:
```

Comment:
so that it is somewhere, noop tracer do not have propagators so we need a try catch here. The requests opencensus extension does the same thing.

https://github.com/census-instrumentation/opencensus-python/blob/master/contrib/opencensus-ext-requests/opencensus/ext/requests/trace.py#L115-L121